src/dom/rect.js

```diff
@@ -40,6 +40,20 @@ export default class Rect {
 	 * @param {HTMLElement|Range|ClientRect|module:utils/dom/rect~Rect|Object} obj A source object to create the rect.
 	 */
 	constructor( obj ) {
+		/**
+		 * The object this rect is for.
+		 *
+		 * @protected
+		 * @readonly
+		 * @member {HTMLElement|Range|ClientRect|module:utils/dom/rect~Rect|Object} #_obj
+		 */
+		Object.defineProperty( this, '_obj', {
```

Comment:
`defineProperty` to set `enumerable` `false`. Without it iterating over the `Rect` would also return `_obj` and make some pieces of code more complex.

As for `_obj`, maybe not `_context` because it makes people think about `this` and stuff but `_from`?